src/app/credExplorer/App.js

```diff
@@ -65,15 +64,13 @@ export class App extends React.Component<Props, State> {
     const {graphWithAdapters, pnd} = this.state.data;
     return (
       <div style={{maxWidth: "66em", margin: "0 auto", padding: "0 10px"}}>
-        <header className={css(styles.header)}>
-          <h1>Cred Explorer</h1>
-        </header>
         <div>
           <RepositorySelect
             localStore={localStore}
             onChange={(selectedRepo) => this.setState({selectedRepo})}
           />
           <br />
+          <br />
```

Comment:
Eh… can we achieve the same result without the hacks by, making
`RepositorySelect` return a `<p>{whateverItReturnsNow()}</p>`?